File: shoring/flywheel/src/test/java/com/panopset/tests/TestDirectory.java
package com.panopset.tests;

import java.io.File;

import com.panopset.compat.Fileop;

public interface TestDirectory {

	File TEST_DIRECTORY = new File(
			Fileop.combinePaths(Fileop.getCanonicalPath(Fileop.TEMP_DIRECTORY), "/com/panopset/test"));

}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/TestDirectory.kt
package com.panopset.tests

import com.panopset.compat.Fileop
import java.io.File

val TEST_DIRECTORY = File(
    Fileop.combinePaths(Fileop.getCanonicalPath(Fileop.TEMP_DIRECTORY), "/com/panopset/test")
)
